From: Antonio Radici <antonio@debian.org>
Date: Sun, 28 Aug 2016 19:15:06 +0100
Subject: fix_spelling_errors

Fix a small spelling error
---
 postgrey | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/postgrey
+++ b/postgrey
@@ -980,7 +980,7 @@
 
 =item *
 
-At least 5 successfull attempts of delivering a mail (after greylisting was
+At least 5 successful attempts of delivering a mail (after greylisting was
 done). That number can be changed by specifying a number after the
 --auto-whitelist-clients argument. Only one attempt per hour counts.
 
